### ISSUE TITLE
Anchor Money Market on Bombay

### ISSUE THREAD

**Issue by 0xkyro** [Anchor-Protocol/money-market-contracts#38]
Saw that you guys are hard at work with the contracts, great job! Can see that many things are still in progress.

We are thinking of doing some integration testing with Anchor money market. Was wondering is there already a deployed version on `bombay-10`? Will the contract be the same address as the one mentioned on your docs?

**Comment by MSNTCS** [comment]
@0xkyro Hello there, anchor contracts were migrated on `bombay-10`, the address is the same as anchor `tequila-0004`/testnet contracts. Thanks for your concerns.

**Comment by curest0x1021** [comment]
`Bombay-11` was launched yesterday and I found a bug on Anchor Money Market.
When I tried to query for getting Anchor MoneyMarket Contract config, it returns 500 error.

MoneyMarket Contract Address in Bombay-11:
https://finder.terra.money/bombay-11/address/terra15dwd5mj8v59wpj0wvt233mf5efdff808c5tkal 1
query: {“config”: {}}

Expected result: Returns the Anchor MoneyMarket Contract configuration

Current status: It returns status code 500

I tried to query on tequila-004 testnet and columbus-4 mainnet, and it worked well.
Also this query was working well on Bombay-10 testnet, but doesn’t work after launched Bombay-11 .
How can I solve this problem? Let me know your suggestion.
